orion/controllers/announce/announceController.go

```diff
@@ -0,0 +1,107 @@
+package announce
+
+import (
+	"errors"
+	"net/http"
+	"regexp"
+	
+	"github.com/gin-gonic/gin"
+)
+
+const REGEX = `[A-Za-z]+`
```

Comment:
can you give a more meaningful REGEX here and a comment of what it is trying to regex?
REGEX_MESSAGE and REGEX_AUTHOR?
REGEX_LETTERS_ONLY?
REGEX_NON_EMPTY?